lib/src/fsmonitor.rs

```diff
@@ -24,6 +24,7 @@ use std::path::PathBuf;
 use std::str::FromStr;
 
 /// The recognized kinds of filesystem monitors.
+#[derive(PartialEq)]
```

Comment:
nit: add `Eq` too?

Reply:
Seems like a reasonable thing and a good catch. Fixed.